test-harness/index.ts

```diff
@@ -1,20 +1,24 @@
-import { parseSpecFile } from './helpers';
+import { ChildProcess, spawn, spawnSync } from 'child_process';
 import * as fs from 'fs';
+import { validate } from 'gavel';
+import * as globFs from 'glob-fs';
+import { parseResponse } from 'http-string-parser';
 import * as os from 'os';
 import * as path from 'path';
-import * as tmp from 'tmp';
-import { ChildProcess, spawnSync, spawn } from 'child_process';
 import * as split2 from 'split2';
-const validate = require('gavel').validate;
-import { parseResponse } from 'http-string-parser';
+import * as tmp from 'tmp';
+import { parseSpecFile } from './helpers';
 
-jest.setTimeout(60000);
+const glob = globFs({ gitignore: true });
+jest.setTimeout(5000);
 
 describe('harness', () => {
-  const files = fs.readdirSync(path.join(__dirname, './specs/'));
+  const files = process.env.TESTS
```

Comment:
Yes, or multiple test files separated by a comma.

Reply:
Ok nice, can you update the documentation? Test Harness has a readme.md file — it would be cool to put both the env variable as well as the glob rules so we know how to use this stuff.

Reply:
done